Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -649,6 +649,7 @@ var TextInput = React.createClass({
           selectTextOnFocus={this.props.selectTextOnFocus}
           onLayout={this.props.onLayout}
           placeholder={this.props.placeholder}
+          placeholderTextColor={this.props.placeholderTextColor}
```

Comment:
This file is named XXX.windows.js but it contains references to Android and IOS... The AndroidTextInput is also referenced and properties set on it... It seems that anything not related to windows or wpf should be removed from this file... 

Additionally, the code starting on line 635 appears to be related to the PasswordBox control used for password text input... If the boolean secureTextEntry is true the PasswordBoxWindows object is created... Else then the RCTTextInput is created... The Password related code does not appear to be applied if secureTextEntry ... Instead that case is handled by the PasswordBoxWindows.js...